### ISSUE TITLE
Still doesn't work with symbols

### ISSUE THREAD

**Issue by jenbailey** [kenmoore/sketch-relabel-button#49]
Using Sketch 53.2 and relabel button 1.8 - trying to use on button symbol and it won't work. Plugin menu flashes when I hit command J but then sadly nothing else.

**Comment by kenmoore** [comment]
Hi Jen, the latest version of the plugin is 1.10 -- could you please update and let me know if it's not working? Thanks

**Comment by jenbailey** [comment]
YES!!!!! That did the trick! I am such a happy camper. Thank you soooo much. This plugin rules.

**Comment by tbnv** [comment]
It's not working for me. Background is not resized.
